test/index.ts

```diff
@@ -921,29 +992,33 @@ describe('Logging', () => {
       it('should execute callback with error', done => {
         const error = new Error('Error.');
 
-        logging.api[CONFIG.client][CONFIG.method] = (...args) => {
-          const callback = args.pop();
+        logging.api[CONFIG.client][CONFIG.method] = (...args: []) => {
+          // tslint:disable-next-line no-any variable-callback
+          const callback: any = args.pop();
           callback(error);
         };
 
-        logging.request(CONFIG, err => {
+        logging.request(CONFIG, (err: Error) => {
           assert.strictEqual(err, error);
           done();
         });
       });
 
       it('should execute the request function', () => {
-        logging.api[CONFIG.client][CONFIG.method] = (done, ...args) => {
-          const callback = args.pop();
-          callback(null, done);  // so it ends the test
-        };
+        logging.api[CONFIG.client][CONFIG.method] =
+            (done: Function, ...args: []) => {
+              // tslint:disable-next-line no-any variable-callback
+              const callback: any = args.pop();
+              callback(null, done);  // so it ends the test
+            };
 
         logging.request(CONFIG, assert.ifError);
       });
     });
 
     describe('makeRequestStream', () => {
-      let GAX_STREAM;
+      // tslint:disable-next-line no-any variable-GAX_STREAM
+      let GAX_STREAM: {cancel: Function}|any;
```

Comment:
if `any` is used, other type declaration is not required as it will be included in `any`

Reply:
Resolved!